### ISSUE TITLE
Periodically refresh "lists-to-serve" without a restart

### ISSUE THREAD

**Issue by groovecoder** [mozilla-services/shavar#89]
See https://github.com/mozilla-services/shavar-server-list-config/pull/2#issuecomment-279812106 ...

We want the server to periodically refresh that list and dynamically update as a result of changes in that bucket.

**Comment by ckolos** [comment]
I don't think this is actually working properly as restarts are still needed for new lists to come online.
